docs/semgrep-app/scm.md

```diff
@@ -58,6 +67,20 @@ Integrate Semgrep into these custom source code management (SCM) tools by follow
     3. Semgrep App detects repositories from your SCM integration.
 10. Add a new repository by following the steps in [Adding a project](../getting-started-with-semgrep-app/#adding-a-project).
 
+## Receiving PR or MR comments in your VPN or on-premise SCM
+
+PR or MR comments are comments or suggestions made by Semgrep App in your SCM tool. These comments provide a description of the issue detected by Semgrep and may offer possible solutions. These comments are a means for security teams (or any team responsible for creating standards) to help their fellow developers write safe and standards-compliant code.

```

Comment:
(Not blocking) I am thinking that maybe we can expand the abbreviation here again? I know it is expanded above, but I am not sure whether the readers will not scroll down to find this and then search for the term SCM in Google and find Supply Chain Management. Feel free to ignore this now...

Reply:
Done.